app/src/main/java/nyc/c4q/jonathancolon/inContaq/contactlist/fragments/ContactStatsFragment.java

```diff
@@ -65,9 +64,15 @@
     private static final int DEC = 12;
     private static final int DEFAULT_VALUE = 0;
     private final String TAG = "sms";
-
     int highestValue;
+    /*
+    These color constants are hardcoded for backwards compatibility.
+     */
+    private Contact contact;
     private LineChartView lineGraph;
+    private Spinner dateSpinner;
+    private ArrayAdapter<CharSequence> spinnerArrayAdapter;
```

Comment:
spinnerAdapter

No need to specify implementation in the name. Just say what it does, not how it does it.